README.md

```diff
@@ -163,7 +163,7 @@ In alphabetical order:
 + Sahagun, Manny <msahagun@conversantmedia.com>
 + Tiffany Wu <twu@triplelift.com>
 + Vladimir Marteev <xor104@gmail.com>
-+ astudnicky <astudnicky@sonobi.com>
```

Comment:
Happy to add your name to the list, but please don't remove the names of other contributors.  This is taken from the git history.

Reply:
He is no longer with our company.